rnd/autogpt_builder/src/app/login/page.tsx

```diff
@@ -195,8 +195,8 @@ export default function LoginPage() {
               </Button>
             </div>
           </form>
-          <p className="text-red-500 text-sm">{feedback}</p>
-          <p className="text-primary text-center text-sm">
+          <p className="text-sm text-red-500">{feedback}</p>
```

Comment:
Hey there since you are here ;)

```suggestion
          <p className="text-sm text-destructive">{feedback}</p>
```